plugins/modules/terraform.py

```diff
@@ -581,7 +581,9 @@ def main():
         command.append('-parallelism=%d' % module.params.get('parallelism'))
 
     def format_args(vars):
-        if isinstance(vars, str):
+        if isinstance(vars, type(None)):
```

Comment:
The standard `None` check is:
```suggestion
        if vars is None:
```